packages/scripts/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@omakase/scripts",
+  "version": "0.0.1",
+  "description": "A collection of dependencies commonly used across projects",
+  "repository": "https://github.com/omakase-js/omakase/tree/master/packages/scripts",
+  "author": "Artsy Inc.",
+  "license": "MIT",
+  "dependencies": {
+    "@artsy/auto-config": "^1.0.0",
```

Comment:
I don't really know that `auto-config` should go in here. I'm working on a PR to remove the need to install it at all (based on some work that I'm doing in [autobot](https://github.com/auto-it/autobot)).

It's very much a build system thing though.